devimint/src/tests.rs

```diff
@@ -234,6 +234,27 @@ pub async fn latency_tests(dev_fed: DevFed) -> Result<()> {
         fm_internal_pay.push(start_time.elapsed());
     }
 
+    info!("Waiting for a new session");
+    let session_count = cmd!(client, "dev", "api", "session_count")
+        .out_json()
+        .await?["value"]
+        .as_u64()
+        .context("session count must be integer")?
+        .to_owned();
+    let start = Instant::now();
+    poll("Waiting for a new session", 6, || async {
+        info!("Awaiting session outcome {session_count}");
+        match cmd!(client, "dev", "api", "await_session_outcome", session_count)
+            .run()
+            .await
+        {
+            Err(e) => Err(ControlFlow::Continue(e)),
+            Ok(_) => Ok(()),
+        }
+    })
+    .await?;
+    let session_found_in = start.elapsed();
+
```

Comment:
I don't think this measures what you expect? If you are *somewhere* within a session it could take any time between 0s-3m to finish it. If you want to measure session completion time you have to start measuring at the start of a session.

Reply:
it is not measuring session completion, it is just an assertion that a session completed

Reply:
If you display the time in the output it does kinda looks like you were measuring it... :D

Reply:
This test could have a concurrent task that just measures session durations in parallel with the rest of the test.

Reply:
i have removed this commit for now, lets do it in a separate PR

Reply:
Tracked in https://github.com/fedimint/fedimint/issues/4208